plugin/bundles/serply/plugins/video_search/plugin.py

```diff
@@ -0,0 +1,31 @@
+import json
+from aiohttp import ClientSession
+
+
+from bundle_dependency import *
+from config import CONFIG
+
+
+class VideoSearch(PluginHandler):
+    async def execute(self, credentials: BundleCredentials, plugin_input: PluginInput) -> PluginOutput:
+        query: str = plugin_input.input_params.get("query")
+        count: float = plugin_input.input_params.get("count", None)
```

Comment:
double check if type float is necessary, and change to int if float is not necessary.